float32 angle
float32 distance
---
obstacle_msg data
